src/uct/ib/mlx5/gdaki/gdaki.c

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2025. ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "gdaki.h"
+
+#include <ucs/time/time.h>
+#include <ucs/datastruct/string_buffer.h>
+#include <uct/ib/mlx5/rc/rc_mlx5.h>
+
+#include <doca_log.h>
+
+typedef struct {
+    uct_rc_iface_common_config_t super;
+    uct_rc_mlx5_iface_common_config_t mlx5;
+} uct_gdaki_iface_config_t;
```

Comment:
IMO we should call the new transport "rc_gdaki" because we may also want to add "dc_gdaki"